lint-staged.config.js

```diff
@@ -1,3 +1,3 @@
 export default {
-  '*.ts*': ['npm run format --', 'npm run lint --'],
+  '*.ts*': ['npm run build . --', 'npm run format --', 'npm run lint --'],
```

Comment:
Without the `.` here vite build kept failing out with the error `Could not resolve entry module "vite.config.ts/index.html"`

Reply:
Adding the vite build step here so that in the future when we set cycle detection to return a nonzero exit code via `circleImportThrowErr`, this will result in adding a new import cycle to break pre-commit linting